projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -643,6 +643,7 @@ public static function enqueue_block_editor_assets() {
 		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
 			$user                      = wp_get_current_user();
 			$user_data                 = array(
+				'email'    => $user->user_email,
```

Comment:
Is it worth limiting to add this only when Email/subscribe features are active?

Reply:
I wouldn't think so, to be honest. Some of the other items we populate in that initial state are only used by a few specific blocks, but we do not necessarily limit them to specific modules. 
And since we request user information anyway, adding one more item to the array doesn't seem too expensive to me.